dhall-docs/README.md

```diff
@@ -9,3 +9,18 @@ For installation or development instructions, see:
 This `dhall-docs` package provides a cli utility that takes a dhall package or file and outputs
 a HTML documentation of it.
 
+## Usage
+
+The easiest usage is the following:
+
+```bash
+dhall-docs --input <YOUR-PACKAGE-FOLDER>
```

Comment:
I like to use Bash variables as placeholders for user-supplied command-line values in documentation:

```suggestion
dhall-docs --input "${YOUR-PACKAGE-FOLDER}"
```